sw/device/silicon_creator/lib/boot_svc/boot_svc_primary_bl0_slot.h

```diff
@@ -58,15 +58,21 @@ typedef struct boot_svc_primary_bl0_slot_res {
    */
   boot_svc_header_t header;
   /**
-   * Response from ROM_EXT.
+   * New primary slot set by the boot service.
+   */
+  uint32_t new_primary_bl0_slot;
```

Comment:
nit (just for the sake of consistency with the request struct):

```suggestion
   * New primary BL0 slot set by the boot service.
   */
  uint32_t primary_bl0_slot;
```